doc/manuals/admin/management_api.md

```diff
@@ -28,3 +29,67 @@ curl --request PUT <host>:<port>/log/trace/t1-t2,t3-t4
 'PUT-requests' overwrite the previous log settings. So, in order to ADD
 a trace level, a GET /log/trace must be issued first and after that the
 complete trace string to be sent in the PUT request can be assembled.
+
+
+## Sempahores
+Another useful (especially if the broker stops responding correctly) REST API is
+the semaphore listing offered:
+
+
+```
+curl <host>:<port>/admin/sem
+```
+
+The response is a listing of information of all the broker's semaphores:
+
+```
+{
+    "alarmMgr": {
+        "status": "free"
+    },
+    "connectionContext": {
+        "status": "free"
+    },
+    "connectionSubContext": {
+        "status": "free"
+    },
+    "dbConnection": {
+        "status": "free"
+    },
+    "dbConnectionPool": {
+        "status": "free"
+    },
+    "logMsg": {
+        "status": "free"
+    },
+    "request": {
+        "status": "free"
+    },
+    "subCache": {
+        "status": "free"
+    },
+    "timeStat": {
+        "status": "free"
+    },
+    "transaction": {
+        "status": "free"
+    }
+}
+```
+
+Short explanation of the semaphores:
+* alarmMgr               Protects the data of the Alarm Manager 
+* connectionContext      Protects the curl context for sending HTTP notifications/forwarded messages
+* connectionSubContext   Protects the curl context for sending HTTP notifications/forwarded messages
+* dbConnectionPool       Protects mongo connection pool
+* dbConnection           Protects the set of connections of the mongo connection pool
+* logMsg                 Makes sure that not two messages are written simultaneously to the log-file
```

Comment:
Inside the functest it is always 'free'. I ran the functest 25 times.
Manually, I got 'taken' something like 5 out of 5 times. I have no explanation for it.
We might need a REGEX((free|taken)) in the future, we'll see ...